metron-stellar/stellar-common/src/test/java/org/apache/metron/stellar/dsl/functions/RegExFunctionsTest.java

```diff
@@ -68,4 +68,19 @@ public void testRegExGroupVal() throws Exception {
       Assert.assertTrue("Did not fail on wrong number of parameters",false);
     }
   }
+
+  @Test
+  public void testRegExReplace() throws Exception {
+    final Map<String, String> variableMap = new HashMap<String, String>() {{
+      put("numbers", "12345");
+      put("numberPattern", "\\d(\\d)(\\d).*");
+      put("letters", "abcde");
+      put("empty", "");
+    }};
+
```

Comment:
Can you add tests for passing in empty or null strings?

Reply:
Indeed, more tests included.